ui/src/app/cluster-workflow-templates/cluster-workflow-template-list.tsx

```diff
@@ -51,6 +52,8 @@ export function ClusterWorkflowTemplateList({history, location}: RouteComponentP
 
     useCollectEvent('openedClusterWorkflowTemplateList');
 
+    const [storedDisplayISOFormat, setStoredDisplayISOFormat] = useTimestamp(TIMESTAMP_KEYS.CLUSTER_WORKFLOW_TEMPLATE_LIST);
```

Comment:
similarly, the existing style has all `const`s at the top of the component, while this is an outlier. please follow the existing style